### ISSUE TITLE
Issues running `wal -i`

### ISSUE THREAD

**Issue by crinklywrappr** [elves/elvish#1380]
First of all, Elvish has reached a point where I can use it as my primary shell.  Huge thanks to @xiaq for all your hard work!

## The problem
pywal generates and applies color schemes to various applications based on a picture.  When running `wal -i` from inside elvish, terminals occasionally stop accepting keyboard input.  I have tried this in both `kitty` and `gnome-terminal`.  Kitty also displays a random character in the prompt, which you can see in the screenshot.  This works fine in bash.

![Screenshot from 2021-08-13 16-35-28](https://user-images.githubusercontent.com/56522/129421964-c7f3512d-2815-4b9f-8c2a-b061b355248f.png)

## Technology used
Check the `neofetch` output in the screenshot for lots of information.

EDIT: I reproduced this in `elvisn-HEAD` on 8/15/21

**Comment by krader1961** [comment]
Does that command run anything in the background? Since Elvish doesn't implement POSIX job control that can cause problems. Does the "random character" persist if you press Enter to get a new prompt? In your screen shot it appears to be the letter `m`. Is it always that letter? Also, it might be useful to first run the `script` command, then `wal -i`, then exit the subshell and examine the resulting *typescript* file.

**Comment by krader1961** [comment]
Also, as I noted in the other issue you opened you should try to reproduce the problem using Elvish built from the current master branch. The 0.15.0 release isn't particularly old chronologically but in terms of the number of commits merged since that release it is ancient.

**Comment by crinklywrappr** [comment]
Maybe this will answer some of your questions

![Screenshot from 2021-08-13 18-16-09](https://user-images.githubusercontent.com/56522/129426556-7e9e5d92-bbcf-4d51-96dc-5bac9c664379.png)

> Does the "random character" persist if you press Enter to get a new prompt?
In the event that I can, yes.

> In your screen shot it appears to be the letter m. Is it always that letter?
No.  It varies.  One time I got `tt`.

[typescript file](https://gist.github.com/crinklywrappr/cfdfcdbe781d8d9a2847b327b5e953f4) - In this session, it caused the terminal to stop accepting input, so I had to exit uncleanly.

I am certainly willing to try building and running `master`, but I will either need to find some time to set up a VM, or get some instructions on building w/ nix.
